test/functional/test_transformer_implementation.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import numpy as np
+
+from sagemaker_containers import content_types, env, serializers, status_codes, transformers, worker
+import test
+from test import miniml
+
+
+class MiniMlTransformer(transformers.BaseTransformer):
```

Comment:
MiniMl?

Reply:
Is just a fake framework for the tests. @nadiaya and @laurenyu suggested me to not user Keras for the tests.